docs/sdk/src/polkadot_sdk/xcm.rs

```diff
@@ -1,5 +1,69 @@
 //! # XCM
 //!
-//! @KiChjang @franciscoaguirre
-//! TODO: RFCs, xcm-spec, the future of the repo, minimal example perhaps, forward to where actual
-//! docs are hosted. <https://github.com/paritytech/polkadot-sdk-docs/issues/58>
+//! XCM, or Cross-Consensus Messaging, is a **language** to communicate **intentions** between
+//! **consensus systems**.
+//!
+//! ## Overview
+//!
+//! XCM is a standard, whose specification lives in the [xcm format repo](https://github.com/paritytech/xcm-format).
+//! It's agnostic both in programming language and blockchain platform, which means it could be used
+//! in Rust in Polkadot, or in Go or C++ in any other platform like Cosmos or Ethereum.
+//!
+//! It enables different consensus systems to communicate with each other in an expressive manner.
+//! Consensus systems include blockchains, smart contracts, and any other state machine that
+//! achieves consensus in some way.
+//!
+//! XCM is based on a virtual machine, the XCVM.
+//! An XCM program is a series of instructions, which get executed one after the other by the
+//! virtual machine. These instructions aim to encompass all major things users typically do in
+//! consensus systems. There are instructions on asset transferring, teleporting, locking, among
+//! others. New instructions are added via the [RFC process](https://github.com/paritytech/xcm-format/blob/master/proposals/0032-process.md).
```

Comment:
I don't think we should put the focus entirely on just new instructions; we should include new operations of the VM as well:

```suggestion
//! others. New instructions and new operations for the XCVM are added via the [RFC process](https://github.com/paritytech/xcm-format/blob/master/proposals/0032-process.md).
```